modules/ROOT/pages/exchange-integration.adoc

```diff
@@ -11,6 +11,18 @@ The integration with Exchange 2.0 only works for Mavenized projects.
 
 By default, Studio 6.4 points to the Exchange 2.0 URL +https://anypoint.mulesoft.com/exchange2/+.
 
+== Before You Begin
+
+Because Exchange 2.0 provides its own repository and versioning implementation, every new asset that you publish to Exchange 2.0 gets a new Group ID, Asset ID, and Version (GAV) combination.
+
+If you are publishing an asset that already has a GAV combination configured, those Maven coordinates will be overwritten by the new GAV combination when publishing. To avoid messing up your dependencies, you have two options:
```

Comment:
I am a little confused about this comment. If you re-publish an asset in Exchange 2.0 the asset won't be overwritten. Are you talking about having the asset published in another repository?

Reply:
I meant publishing an asset to Exchange 2.0 for the first time that either came from Exchange 1, or  another Maven repository.

Reply:
ok, great, maybe can we add this comment in the documentation?